src/ui/view-subscribe.ts

```diff
@@ -0,0 +1,36 @@
+import { property, html, customElement, css } from 'lit-element'
+import { Connected, State, MyModelSelectors } from './connected'
+import { sharedStyles } from './shared-styles'
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'view-subscribe': ViewSubscribeElement
+  }
+}
+
+@customElement('view-subscribe')
+export class ViewSubscribeElement extends Connected {
+  @property({ type: String })
+  myprop: ""
+
+  mapState(state: State) {
+    return {
+      myprop: MyModelSelectors.squaredValue(state),
+    }
+  }
+
+  render() {
+    return html `<p>From the dispatch Web Component: ${this.myprop}</p>`
+  }
+
+  static get styles() {
+    return [
+      sharedStyles,
+      css`
+        p {
+          font-size: 12px;
+        }
+      `,
+    ]
+  }
+}
```

Comment:
Again, to show the core pieces of rdx / state management if the intention is to provide something for someone to learn, I'd try to keep elements focused on that with anything extra removed.